include/meta/meta.hpp

```diff
@@ -3286,7 +3286,7 @@ namespace meta
                 bind_back<quote_trait<subst1_>, back<As>, drop_c<Ts, size<As>{} - 2>>>>,
             list<back<As>>>;
 
-#ifdef META_CONCEPT
+#if 0//def META_CONCEPT
         template <List As, List Ts>
         requires (_v<size<Ts>> + 2 >= _v<size<As>>)
         using substitutions = substitutions_<As, Ts>;
```

Comment:
This definition of `substitutions` in particular is what is giving gcc heartburn. I didn't investigate deeply.

The other problems below seem to be that gcc is ignoring some `requires` clauses for some reason. Also not investigated.